litgpt/deploy/serve.py

```diff
@@ -0,0 +1,133 @@
+from pathlib import Path
+from typing import Dict, Any, Optional
+from litgpt.utils import check_valid_checkpoint_dir
+
+import lightning as L
+import torch
+from litserve import LitAPI, LitServer
+
+from litgpt.model import GPT
+from litgpt.config import Config
+from litgpt.tokenizer import Tokenizer
+from litgpt.generate.base import generate
+from litgpt.prompts import load_prompt_style, has_prompt_style, PromptStyle
+from litgpt.utils import load_checkpoint, CLI
+
+
+class SimpleLitAPI(LitAPI):
+
+    def __init__(self,
+                 checkpoint_dir: Path,
+                 precision: Optional[str] = None,
+                 temperature: float = 0.8,
+                 top_k: int = 200,
+                 max_generated_tokens: int = 30) -> None:
```

Comment:
This is a pretty small number, I'm wondering if users would get confused by such a default.

I think I know where this comes from (Phi as the example model) but let's change the model rather than kneecapping generation.

Reply:
That's fair. We have 50 in `litgpt generate`, maybe we should make it at least consistent or update in both places?

Reply:
I'm fine to standardize on 50. I'm seeing 64 or 128 in other packages (e.g. ollama has 128 if I'm not mistaken).